tests/profiler/tensorflow2/test_native_tf2_profiler.py

```diff
@@ -31,65 +34,52 @@
 
 
 @pytest.fixture
-def native_tf2_cprofile_profiler_config_parser(config_folder, monkeypatch):
-    config_path = os.path.join(
-        config_folder, "test_native_tf2_cprofile_profiler_config_parser.json"
-    )
-    monkeypatch.setenv("SMPROFILER_CONFIG_PATH", config_path)
-    return ProfilerConfigParser(FRAMEWORK.TENSORFLOW)
+def native_tf2_cprofile_profiler_config_path(config_folder, monkeypatch):
+    return os.path.join(config_folder, "test_native_tf2_cprofile_profiler_config_parser.json")
 
 
 @pytest.fixture
-def native_tf2_pyinstrument_profiler_config_parser(config_folder, monkeypatch):
-    config_path = os.path.join(
-        config_folder, "test_native_tf2_pyinstrument_profiler_config_parser.json"
-    )
+def native_tf2_pyinstrument_profiler_config_path(config_folder, monkeypatch):
+    return os.path.join(config_folder, "test_native_tf2_pyinstrument_profiler_config_parser.json")
     monkeypatch.setenv("SMPROFILER_CONFIG_PATH", config_path)
     return ProfilerConfigParser(FRAMEWORK.TENSORFLOW)
 
```

Comment:
Are there two return statements in this function?

Reply:
Thanks for catching, forgot to revert these changes.